test.sh

```diff
@@ -0,0 +1,50 @@
+#/bin/bash
```

Comment:
Do we want to keep this file? If so, might be good to move it inside a scripts directory.